### PR TITLE
parametrize package names and use akka 2.5.10

### PR DESCRIPTION
**Pull request by shkoder**
* parametrize package names with $package$
     * use akka version 2.5.10 by default
     * fix formatting issue
     * fix regex in docs for removing comments

### CODE REVIEW

**Comment by 2m** [comment]
Package names should only be parametrised under the `src/main/g8` directory. That is the source for the gitter8 template.

Other sources (like under `docs/src/main/paradox/`) are part of the guide that is hosted [here](https://developer.lightbend.com/guides/akka-http-quickstart-scala/). So it does not really make sense to parametrise package names there.

File: src/main/g8/src/main/scala/$package$/UserRegistryActor.scala [hunk 1]
```diff
@@ -1,7 +1,8 @@
-package com.lightbend.akka.http.sample
+package $package$
 
 import akka.actor.{ Actor, ActorLogging, Props }
 
+//#user-registry-actor
```
Comment (raboof) [inline src/main/g8/src/main/scala/$package$/UserRegistryActor.scala]: Might be good to include the imports in the snippet?

**Comment by shkoder** [comment]
@2m for the package parametrisation to work I've actually moved .scala files from 
`src/main/g8/src/main/scala/com/lightbend/akka/http/sample/`
to
`src/main/g8/src/main/scala/$package$/`
That's why `docs/src/main/paradox/*` need to have their references updated.

File: src/main/g8/src/main/scala/$package$/UserRegistryActor.scala (hunk 1 above)
Comment (shkoder) [reply src/main/g8/src/main/scala/$package$/UserRegistryActor.scala]: good point, addressed

**Comment by 2m** [comment]
Ahh. I misunderstood it previously. LGTM then!

**Comment by shkoder** [comment]
@patriknw creating a project from https://developer.lightbend.com/start/?group=akka&project=akka-http-quickstart-scala works as expected

**Pull request merged**